sdcm/nemesis.py

```diff
@@ -2090,10 +2090,15 @@ def call_next_nemesis(self):
         self.execute_disrupt_method(disrupt_method=next(self.disruptions_cycle))
 
     @latency_calculator_decorator(legend="Run repair process with nodetool repair")
-    def repair_nodetool_repair(self, node=None, publish_event=True):
-        node = node if node else self.target_node
-        with adaptive_timeout(Operations.REPAIR, node, timeout=HOUR_IN_SEC * 48):
-            node.run_nodetool(sub_cmd="repair", publish_event=publish_event)
+    def repair_nodetool_repair(self, node=None, publish_event=True, all_nodes=False):
+        if all_nodes:
+            for node in self.cluster.nodes:
```

Comment:
you can use only data nodes `self.clsuter.data_nodes`. Repair operation could be skipped on zero-token nodes